.github/workflows/build.yml

```diff
@@ -6,120 +6,59 @@ jobs:
-            qt5-default \
-            qt5-gtk-platformtheme \
-            qttranslations5-l10n \
-            libqt5svg5-dev \
-            libboost-dev \
-            libpulse-dev \
-            portaudio19-dev \
-            liblog4cpp5-dev \
-            gnuradio-dev \
-            libairspy-dev \
-            libairspyhf-dev \
-            libfreesrp-dev \
-            libhackrf-dev \
-            libusb-1.0-0-dev \
-            libsoapysdr-dev \
-            soapysdr-module-remote \
-            libuhd-dev \
-            liborc-0.4-dev \
-            libhidapi-dev
-
-          cd /tmp
-          git clone https://gitea.osmocom.org/sdr/rtl-sdr.git
-          cd rtl-sdr
-          mkdir build
-          cd build
-          cmake -DCMAKE_INSTALL_PREFIX=/usr -DDETACH_KERNEL_DRIVER=ON ..
-          make -j4
-          make install
-          ldconfig
-
-          cd /tmp
-          git clone https://github.com/Nuand/bladeRF.git
-          cd bladeRF
-          mkdir build
-          cd build
-          cmake -DCMAKE_INSTALL_PREFIX=/usr ..
-          make -j4
-          make install
-          ldconfig
-
-          cd /tmp
-          git clone https://github.com/dl1ksv/gr-fcdproplus.git
-          cd gr-fcdproplus
-          mkdir build
-          cd build
-          cmake -DCMAKE_INSTALL_PREFIX=/usr ..
-          make -j4
-          make install
-          ldconfig
-
-          cd /tmp
-          git clone https://gitea.osmocom.org/sdr/libosmo-dsp.git
-          cd libosmo-dsp
-          autoreconf -i
-          ./configure --prefix=/usr
-          make -j4
-          make install
-          ldconfig
+          sudo apt-get update -qq
+          sudo apt-get install -y --no-install-recommends \
+            libegl1
 
-          cd /tmp
-          git clone https://gitea.osmocom.org/sdr/gr-iqbal.git
-          cd gr-iqbal
-          git checkout gr3.8
-          mkdir build
-          cd build
-          cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_MODULES_DIR=/usr/lib/x86_64-linux-gnu/cmake ..
-          make -j4
-          make install
-          ldconfig
-
-          cd /tmp
-          git clone https://github.com/gqrx-sdr/gr-osmosdr.git
-          cd gr-osmosdr
-          git checkout origin/gr3.8
-          mkdir build
-          cd build
-          cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_MODULES_DIR=/usr/lib/x86_64-linux-gnu/cmake -DENABLE_PYTHON=False ..
-          make -j4
-          make install
-          ldconfig
+      - name: Install conda dependencies
+        id: setup-micromamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-name: gqrx
+          create-args: >-
+            c-compiler
+            cxx-compiler
+            cmake
+            make
+            pkg-config
+            boost-cpp
+            gnuradio-core
+            gnuradio-osmosdr
+            mesa-libgl-devel-cos6-x86_64
+            pybind11
+            pulseaudio
+            qt-gtk-platformtheme
+            qt-main
```

Comment:
Gqrx has full support for Qt 6, so these could probably be updated to `qt6-gtk-platformtheme` and `qt6-main`. (I'm not sure whether any further changes would be needed.)

Reply:
If switching to Qt 6 would negatively impact compatibility with old Linux distributions, we could always stick with Qt 5 for now.

Reply:
From inspecting the recipes, it looks like it would still work on old distributions. It's worth a try.

Reply:
Qt 6 is working fine for me! We can squash these commits at an opportune time.